src/stc/scintilla/include/Scintilla.iface

```diff
@@ -331,7 +331,7 @@ set void MarkerSetBack=2042(int markerNumber, colour back)
 # Set the background colour used for a particular marker number when its folding block is selected.
 set void MarkerSetBackSelected=2292(int markerNumber, colour back)
 
-# Enable/disable highlight for current folding bloc (smallest one that contains the caret)
+# Enable/disable highlight for current folding block (smallest one that contains the caret)
```

Comment:
Should we check this file in `misc/scripts/spellcheck`?

Reply:
No, this is a file from Scintilla. I don't think we should spellcheck files from external projects.
You fixed this in `include/stc/stc/h` and `interface/stc/stc.h` when you added spellcheck, but these changes will be overwritten when these files are regenerated.
I think it is better to fix this in the Scintilla source, than to ignore the stc files/lines in spellcheck. It is also fixed in a later version if Scintilla, so it should fix itself once Scintilla get updated.

Reply:
> You fixed this in `include/stc/stc/h` and `interface/stc/stc.h` when you added spellcheck,

Oh, indeed, thanks for noticing this, I totally failed to realize that I was editing a generated file, even if I really should have known better (but actually I didn't edit it, the changes were done by codespell itself, with its `-w` option, which is why I didn't even notice which file was being changed).

> I think it is better to fix this in the Scintilla source, than to ignore the stc files/lines in spellcheck. It is also fixed in a later version if Scintilla, so it should fix itself once Scintilla get updated.

Yes, absolutely, thanks again.